I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 8, in script call
    call ch1_main from _call_ch1_main
  File "game/script-ch1.rpy", line 654, in script
    da "jeg kan godt forstå at mc er din ynglings elev toy freddy"
Exception: Sayer 'da' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 8, in script call
    call ch1_main from _call_ch1_main
  File "game/script-ch1.rpy", line 654, in script
    da "jeg kan godt forstå at mc er din ynglings elev toy freddy"
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 2579, in execute
    Say.execute(self)
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 581, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 470, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'da' is not defined.

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.4.24120703
A Week Of School 2 1.0.0
Mon Dec 16 17:11:04 2024
